app/views/user_mailer/account_activation.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Sample App</h1>
+
+<p>Hi <%= @user.name %>,</p>
```

Comment:
I18n (you can use interpolation to pass in the name)

https://stackoverflow.com/questions/6536304/how-can-i-perform-interpolation-in-i18n-locales